utils/logging/src/main/kotlin/LoggingUtils.kt

```diff
@@ -19,12 +19,25 @@
 
 package org.eclipse.apoapsis.ortserver.utils.logging
 
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.slf4j.MDCContext
 import kotlinx.coroutines.withContext
 
 import org.slf4j.MDC
 
+/**
+ * A wrapper for [kotlinx.coroutines.runBlocking] which always adds an [MDCContext] to the newly created coroutine
+ * context. This ensures that SLF4J's MDC context is not lost when `runBlocking` is used.
+ *
+ * This function should be used instead of [kotlinx.coroutines.runBlocking] in all code to ensure that the MDC context
+ * is always preserved.
+ */
+fun <T> runBlocking(context: CoroutineContext = EmptyCoroutineContext, block: suspend CoroutineScope.() -> T): T =
```

Comment:
I don't want to add a dependency `ort-utils` everywhere because it brings in unrelated dependencies like `software.amazon.awssdk:s3` and `org.apache.commons:commons-compress`.

Reply:
And anyway it just occurred to me that ORT's version adds `org.apache.logging.log4j.kotlin.CoroutineThreadContext`, whereas this adds `kotlinx.coroutines.slf4j.MDCContext`, due to the different logging APIs used in ORT and ORT server. We should probably align that at some point, but that's a different topic.

Reply:
Yes, I think I will change the workers to also to use the ORT Server version for consistency later on. But it still works as explained in the commit message:

```
    Note that the server uses the SLF4J API to access the MDC while ORT uses
    the Log4j API. This still works because with `log4j-to-slf4j` on the
    classpath, Log4j uses SLF4J's MDC implementation.
```